html/demos/streaming.js

```diff
@@ -248,7 +248,16 @@ $(document).ready(function() {
 										}
 									});
 									Janus.attachMediaStream($('#remotevideo' + mid).get(0), stream);
-									$('#remotevideo' + mid).get(0).play();
+									let playPromise = $('#remotevideo' + mid).get(0).play();
+									if (playPromise !== undefined) {
+										playPromise
+											.then(function() {
+												Janus.log('start playing')
+											})
+											.catch(function(error) => {
+												Janus.error('failed to play', error)
```

Comment:
Both the log messages should start with a capital letter (code style).

As a side note, I don't like "start playing" as a message, as it feels like broken English to me. Something like "Starting playout" would probably be better.

Reply:
Thank you for checking.

The result of google search for "Starting playout" is 640.
On the other hand, "Start playing" is 30,500,000.
Is it really better?

I changed to "Started playing".